virt_lightning/__init__.py

```diff
@@ -32,15 +34,21 @@ def __init__(self, configuration, uri="qemu:///session"):
             exit(1)
         self.conn = conn
         self.configuration = configuration
+        self.network = ipaddress.ip_network(
+            self.configuration.get("network", "192.168.122.0/24")
```

Comment:
This line need configurable because some users can use another subnet.

Reply:
This is the reason why I go through the configuration dict. This being said, we need a much better configuration file.

Reply:
c.f: https://github.com/virt-lightning/virt-lightning/issues/8